mycli/myclirc

```diff
@@ -80,6 +80,9 @@ auto_vertical_output = False
 # keyword casing preference. Possible values "lower", "upper", "auto"
 keyword_casing = auto
 
+# disabled pager on startup
+nopager = False
```

Comment:
Let's use the name `enable_pager` and use the `True` and `False` values. 

Also move this option closer to the `pager` option in the config file.

Reply:
for @meeuw ... on my pc i haven't the .my.cnf (i have not a mysql server local)
for @amjith ... good remark ... i work

Reply:
@fraoustin you don't need a mysql server to have `.my.cnf`, `.my.cnf` is in the same directory as `.myclirc`. The advantage of `.my.cnf` is that it's also used for the official mysql client (I tend to use both mycli and the official mysql client)